broker-daemon/block-order-worker/index.js

```diff
@@ -790,6 +831,21 @@ class BlockOrderWorker extends EventEmitter {
       fsm.removeAllListeners()
     })
   }
+
+  /**
+   * Checks if the relayer is available by pinging the healthCheck endpoint. If available,
+   * returns true. False otherwise.
+   * @private
+   * @returns {Promise<boolean>}
+   */
+  async relayerIsAvailable () {
+    try {
+      await this.relayer.adminService.healthCheck({})
```

Comment:
is the only output here a return value or a `throw`? Is there no value that could be returned that would cause us to determine that the Relayer was unavailable?